client/src/constants/validation.js

```diff
@@ -18,8 +18,15 @@ const validatePastDateString = (s) => {
   if (!validateDateString(s)) return false;
   return Date.parse(s) <= new Date();
 };
-const validateDateIsInThePast = (d) => {
-  return Date.parse(d) <= new Date();
+const validateDateIsInThePast = (s) => {
+  return Date.parse(s) <= new Date();
+};
+const validateDateIsReasonable = (d) => {
+  try {
+    return Date.parse(d) >= Date.parse('1899/12/31');
```

Comment:
Nit: It would be nice to store this date as a constant and then it can be used here for comparison as well as in the error message so it doesn't need to be changed in multiple places in the future

(pretty unlikely this date will change though)

Reply:
Do we have a tech debt ticket logged for constants? Let's get this tracked!

Reply:
Tech debt tracked in HCAP 762